docs/guidelines/content-guidelines/06-links.md

```diff
@@ -26,9 +26,9 @@ Within a relative link, it's possible to link to the heading of a document. To d
 
 To add a reference to a YAML, JSON, SVG, PNG, or JPG file located in the `assets` folder in the same topic, use GitHub relative links. For example, write `[Here](./assets/mf-namespaced.yaml) you can find a sample micro frontend entity.` When you click such a link on the `kyma-project.io` website, it opens the file content in the same tab.
 
-## Links in documentation toggles
+## Links in tabs
```

Comment:
Why is the headline about tabs, but the content about (html-based) toggles?

Also, can we link to a docsify tab?

Reply:
I left the original title. Linking in docsify tabs should work without any extra complicity but that we should test.